### PR TITLE
crud users

### PR DESCRIPTION
**Pull request by hoangtv2605**
## Related Tickets:
- [26939](https://edu-redmine.sun-asterisk.vn/issues/26939)
- [26941](https://edu-redmine.sun-asterisk.vn/issues/26941)
- [26942](https://edu-redmine.sun-asterisk.vn/issues/26942)

## WHAT this PR does?
- Dùng modal để create và edit user
- Dùng Datatable để show
- Export list users ra file excel

## Notes Impacted Areas:
- N/A.

### CODE REVIEW

**Comment by hoangtv2605** [comment]
ready

File: app/Http/Controllers/UserController.php [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+
+        $user->update($data);
+
+        return redirect()->route('users.index')->withSuccessTitle(trans('request.success'));
+    }
+
+    public function destroy($id)
+    {
+        try {
+            $user = User::find($id);
```
Comment (huyenttm-0901) [inline app/Http/Controllers/UserController.php]:
```suggestion
            $user = User::findOrFail($id);
```

File: resources/views/exports/users.blade.php [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+            <th>{{ trans('request.id') }}</th>
+            <th>{{ trans('message.name') }}</th>
+            <th>{{ trans('message.email') }}</th>
+            <th>{{ trans('message.phone_number') }}</th>
+            <th>{{ trans('message.created_at') }}</th>
+            <th>{{ trans('message.updated_at') }}</th>
+        </tr>
+    </thead>
+    <tbody>
+        @foreach($users as $key => $user)
```
Comment (huyenttm-0901) [inline resources/views/exports/users.blade.php]: convention

**Comment by hoangtv2605** [comment]
ready

**Comment by hoangtv2605** [comment]
ready

**Pull request merged**